apps/hash-frontend/src/pages/signup.page/account-setup-form.tsx

```diff
@@ -82,319 +137,233 @@ export const AccountSetupForm: FunctionComponent<AccountSetupFormProps> = ({
-            </label>
-            <p
-              style={{
-                color: "#000000",
-                fontSize: "0.875rem",
-                lineHeight: "1.25rem",
-                marginBottom: "1.25rem",
-                opacity: 0.6,
-              }}
-            >
-              What shall we call you when referring to you? e.g. “Hi,{" "}
-              <strong
-                style={{
-                  color: "#000000",
-                  opacity: 1,
-                  textTransform: "capitalize",
-                }}
-              >
-                {preferredNameWatcher || "Bobby"}
-              </strong>
-              ”
-            </p>
-            <Box
-              component="input"
-              id="name"
-              sx={{
-                paddingLeft: "1.25rem",
-                paddingRight: "1.25rem",
-                paddingTop: "1.5rem",
-                paddingBottom: "1.5rem",
-                width: "16rem",
-                height: "2.75rem",
-                borderRadius: "0.5rem",
-                borderColor: "#D1D5DB",
-                borderStyle: "solid",
-
-                "&:focus": {
-                  borderColor: "#2563EB",
-                  outline: "none",
-                },
-              }}
-              placeholder="Bobby"
-              {...register("preferredName", { required: true })}
-            />
-          </div>
-
-          {!!invitationInfo && (
-            <div style={{ marginTop: "2rem" }}>
-              <Controller
-                control={control}
-                name="responsibility"
-                rules={{ required: true }}
-                render={({ field: { onChange, value } }) => (
-                  <SelectInput
-                    className="w-64"
-                    label={`Your Role at ${invitationInfo.orgName}`}
-                    labelClass="font-bold text-base mb-4"
-                    id="responsibility"
-                    options={ORG_ROLES}
-                    onChange={onChange}
-                    value={value}
-                  />
-                )}
-              />
-
-              {errorMessage ? (
-                <p
-                  style={{
-                    marginTop: "1.25rem",
-                    color: "#EF4444",
-                    fontSize: "0.875rem",
-                    lineHeight: "1.25rem",
+                  {displayNameWatcher || "Jonathan Smith"}
+                </Box>
+                ”
+              </>
+            }
+          />
+          <TextField
+            id="name"
+            placeholder="Jonathan Smith"
+            autoFocus
+            sx={{ width: inputWidth }}
+            {...register("displayName", { required: true })}
+          />
+        </Box>
+        <Box>
+          <InputLabel
+            htmlFor="shortname"
+            label={
+              <>
+                Personal Username{" "}
+                <Box
+                  component="span"
+                  sx={{
```

Comment:
This could do with center aligning

<img width="286" alt="Screenshot 2024-02-08 at 09 13 02" src="https://github.com/hashintel/hash/assets/37743469/a3708807-c993-4261-9309-5134323090c3">

Reply:
Have moved the icon one pixel down in https://github.com/hashintel/hash/pull/4009/commits/51dee3de861c9e8f88ab4ac9b50e1058b1e6718b, now looks like this:

![Uploading image.png…]()

Reply:
I'm assuming you can't do flex/align center on the parent for some reason